apps/prairielearn/src/tests/instructorQuestionSettings.test.sql

```diff
@@ -0,0 +1,14 @@
+-- BLOCK update_course_repo
+UPDATE pl_courses
+SET
+  repository = $repo
+WHERE
+  id = 1;
+
+-- BLOCK select_question_by_id
```

Comment:
Note that there's a `selectQuestionById` function in `models/question.ts` that you can use instead.

Reply:
Updated to use `selectQuestionById` function in [4fd6719](https://github.com/PrairieLearn/PrairieLearn/pull/10698/commits/4fd671935566a70b740a03f5534e17952908fb58). Removed the now unused query in [2c9e530](https://github.com/PrairieLearn/PrairieLearn/pull/10698/commits/2c9e5300d7b1e9f7e7e520b2a23c01a2b5b31790).